packages/messages/src/browser/notifications.ts

```diff
@@ -72,8 +74,16 @@ export class Notifications {
                 button.addEventListener('click', () => {
                     action.fn(handler);
                     close();
+                    clearTimeout(this.closeTimer);
                 });
             }
+            if (properties.timeout !== undefined && properties.timeout > 0) {
```

Comment:
minor: `!!properties.timeout && ...` is shorter

Reply:
Done this minor modification